packagedb/api.py

```diff
@@ -522,6 +522,17 @@ def filter_by_checksums(self, request, *args, **kwargs):
 
         qs = Package.objects.filter(q)
         paginated_qs = self.paginate_queryset(qs)
+        enhance_package_data = request.query_params.get('enhance_package_data', False)
```

Comment:
I think the `enhance_package_data` flag should be sent along with the data in the post request, rather than using query_params here

```suggestion
        enhance_package_data = data.get('enhance_package_data', False)
```